vitest.config.ts

```diff
@@ -6,5 +6,6 @@ export default getViteConfig({
   plugins: [tsConfigPaths()],
   test: {
     globals: true,
+    include: ['**/__tests__/**/*.{test,spec}.?(c|m)[jt]s?(x)'],
```

Comment:
What does this do? Why is it required?

Reply:
To prevent vitest run `e2e` tests. So, it will only look to tests inside dir `__tests__`.
We could use `exclude` too, if you prefer, excluding all test cases under `e2e/` and removing the include.

Reply:
![image](https://github.com/user-attachments/assets/5bde8c7d-8804-4195-92d1-9fa1d8de8d82)
Using exclude

Reply:
Just moved it over 👍